src/service/private.overseas.entity.details.ts

```diff
@@ -18,15 +18,40 @@ export const getPrivateOeDetails = async (
     overseasEntityId,
   );
 
+  checkErrorResponse(req, response, overseasEntityId, transactionId, "overseas entity details");
+
+  return response.resource;
+};
+
+export const getBeneficialOwnerPrivateData = async (
+  req: Request,
+  transactionId?: string,
+  overseasEntityId?: string
```

Comment:
Both of these are required by the API, so shouldn't be optional

Reply:
I see you guys were discussing it in UAR-619, so I can prepare the changes and see what you guys decide? @AndreKramer7654 @n-r-b

Reply:
? is not needed here. transactionId and overseasEntityId are needed for the call